src/Rdf/SparqlQuery.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Rdf;
+
+final class SparqlQuery
+{
+    /**
+     * @var string
+     */
+    private $sparql;
+    /**
+     * @var array
+     */
+    private $variables;
+
+    public function __construct(
+        string $sparql,
+        array $variables = []
+    ) {
+        $this->sparql = $sparql;
+        $this->variables = $variables;
```

Comment:
Variables seem to be unused, the todo mentioned it's missing a replace.

Reply:
Yeah! Most of the code in these namespaces has been copied in from the existing OpenSkos, which is littered with _Todos_. We can clean them up as we find them.